css/styles.css

```diff
@@ -74,3 +74,10 @@
   background-color: #e64a19;
   border: 2px solid #e64a19;
 }
+
+.sideBar-icon:hover{
```

Comment:
@moni999fd You should have a custom class like this in a project that uses tailwindcss